README.md

```diff
@@ -1 +1,137 @@
-# first-capstone
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+
+- [🔭 Future Features](#future-features)
+
+- [🤝 Contributing](#contributing)
+
+- [⭐️ Show your support](#support)
+ 
+- [🙏 Acknowledgements](#acknowledgements)
+
+- [📝 License](#license)
+
+
+# 📖  First Capstone : HTML/CSS & JavaScript project <a name="about-project"></a>
+
+**First Capstone: HTML/CSS & JavaScript** is website for IT meeting project build with HTML, CSS, Javascript , using design guidelines , a mobile first approach, with best practices, Github flow and a descriptive and easy to understand README file.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- Linters
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+    <summary>Client</summary>
+        <ul>
+            <li><a  href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+        </ul>
+        <ul>
+            <li><a  href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+        </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+-  Desktop Version
+-  Mobile Version
+-  Home Page
+-  About Page
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://jojo987n.github.io/first-capstone/)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Web browser.
+- Code editor.
+- Git scm.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Run this command: 
+
+```sh
+ cd my-folder
+ git clone git@github.com:jojo987N/first-capstone.git
+```
+### Install
+
+Install this project with:
+
+Run this command:
+
+```sh
+ cd my-folder
+ npm install
+```
+### Usage
+
+To run the project, execute the following command:
+
+Open index.html using live server extension.
+
+### Deployment
+
+- [Live Demo Link](https://jojo987n.github.io/first-capstone/)
+
```

Comment:
- [ ] Documenting your project is an essential part of the development process. That is how you communicate the features of your project with a different set of audience. There was so much background noise on the video you recorded, I could not get any information about the project. I highly suggest you re-record the video to achieve the following guidelines. 🤝️

![Screenshot from 2023-04-19 18-00-40](https://user-images.githubusercontent.com/17243325/233125309-40fbe89e-f063-4117-9e64-34511bb99ac7.png)